frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -222,20 +234,17 @@ impl<T: Config> SignedSubmissions<T> {
 		})
 	}
 
+	/// Remove the signed submission with the highest score from the set.
+	pub fn pop_last(&mut self) -> Option<SignedSubmissionOf<T>> {
+		let best_index = self.indices.len().saturating_sub(1);
```

Comment:
isn't better to return `None` if the `indices == 0`?
It's handled by the `swap_out_submission` anyway so doesn't matter but I would prefer `checked_add` here :)

```suggestion
		let best_index = self.indices.len().checked_sub(1)?;
```